api/v1alpha1/xlinecluster_types.go

```diff
@@ -58,6 +58,9 @@ type XlineClusterSpec struct {
 	// Xline cluster image
 	Image *string `json:"image,omitempty"`
 
+	/// Xline container arguments
+	Args []string `json:"args,omitempty"`
```

Comment:
IMO, define a `BootupArgs` is a better choice in that the kubebuilder can help us validate the field name and its type.